modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc

```diff
@@ -0,0 +1,86 @@
+[id="installing-image-puller-using-che-operator_{context}"]
+= Installing {image-puller-name-short} using the {prod-short} Operator
+
+This section describes how to use the {prod-short} Operator to install the {image-puller-name-short}. This is a Community-Supported technology preview feature.
+
+.Prerequisites
+
+* xref:defining-the-list-of-images-to-pull.adoc[]
+* xref:defining-the-memory-parameters-for-the-image-puller.adoc[]
+ifeval::["{project-context}" == "che"]
+* Operator Lifecycle Manager is available on the {kubernetes} instance.
+endif::[]
+* OperatorHub is available on the OpenShift instance.
+* The {prod-short} Operator is available. See xref:installation-guide:installing-che-on-openshift-4-using-operatorhub.adoc[]
+
+.Procedure
+
+. Edit the `CheCluster` Custom Resource and set `.spec.imagePuller.enable` to `true`
++
+.Enabling {image-puller-name-short} in the `CheCluster` Custom Resource 
+====
+[source,yaml,subs="+attributes"]
+----
+apiVersion: org.eclipse.che/v1
+kind: CheCluster
+metadata:
+  name: {prod-checluster}
+spec:
+  # ...
+  imagePuller:
+    enable: true
+----
+====
++
+[NOTE]
+.Uninstalling {image-puller-name-short} using {prod-short} Operator
+====
+* Edit the `CheCluster` Custom Resource and set `.spec.imagePuller.enable` to `false`:
```

Comment:
@themr0c please backport the example from the original PR - https://github.com/eclipse/che-docs/pull/1800/files#diff-3c2acb4fad07efec5d4153711382a72724f98158b4086d6686b1c7ce461c6b96R83-R97

Reply:
An `.Uninstalling` section makes no sense in modular docs.

Reply:
why?

Reply:
Because the templates are rigid. See: https://redhat-documentation.github.io/modular-docs/#creating-procedure-modules